Documentation/ApiOverview/Logging/Configuration/Index.rst

```diff
@@ -77,13 +77,34 @@ use the following configuration:
 This overwrites the default configuration shown in the first example for classes
 located in the namespace :code:`\Documentation\Examples\Controller`.
 
-For extension "foo" with key "tx_foo" (not using namespaces), the configuration would be located at:
+More examples:
 
 .. code-block:: php
 
-   $GLOBALS['TYPO3_CONF_VARS']['LOG']['Tx']['Foo']['writerConfiguration'] = [
-      // ...
-   ];
+    // configure logging ...
+
+    // for class \Documentation\Examples\Controller\FalExampleController
+    $GLOBALS['TYPO3_CONF_VARS']['LOG']
+        ['Documentation']['Examples']['Controller']['FalExampleController']
+        ['writerConfiguration'] = [
+            // ...
+    ];
+
+    // for channel "security"
+    $GLOBALS['TYPO3_CONF_VARS']['LOG']
+        ['security']
+        ['writerConfiguration'] = [
+             // ...
+    ];
+
+    // For extension "foo" with key "tx_foo" (not using namespaces):
+    $GLOBALS['TYPO3_CONF_VARS']['LOG']
+        ['Tx']['Foo']
+        ['writerConfiguration'] = [
+            // ...
+    ];
+
```

Comment:
```suggestion
    // For extension "foo" with key "tx_my_extension" (not using namespaces):
    $GLOBALS['TYPO3_CONF_VARS']['LOG']
        ['Tx']['MyExtension']
        ['writerConfiguration'] = [
            // ...
    ];

```

`my_extension` is our standard example

Reply:
I am not really familiar with the notation without namespace. If we change it, we should be sure, it is correct.

Should it not be this?

-  // For extension "foo" with key "tx_my_extension" (not using namespaces):
+  // For extension "my_extension" with name "tx_my_extension" (not using namespaces):  

Are we sure, the underscore will not be removed? Anyway, this is not really in the scope of this change (the example already existed before) and we could add the change in a separate PR.

Reply:
I am in favour of leaving this to a separate PR. We should then also check if non-namespaced classes still works this way and if we should still document it (or with a disclaimer).